### PR TITLE
Merging functions to get descriptions from items in game-state/dev to master

### PR DESCRIPTION
**Pull request by jessxia**
This pull request contains the functions to return the short and long descriptions stored in an item struct and their tests (issue #202). This is for WDL's tests of their load_item functions (issue #179). It will also be useful for returning descriptions for actions like LOOK.

### CODE REVIEW

**Comment by a-matai** [comment]
Looks fine—fairly simple PR.

File: src/game-state/tests/test_items.c [hunk 1]
```diff
@@ -48,6 +48,32 @@ Test(item, free)
...
+/* Checks return of short description of item */
+Test(item, get_sdesc_item)
+{
+  item_t *new_item = item_new("item", "short", "long");
+  item_t *null_item = NULL;
+  
+  char *ret = get_sdesc_item(new_item);
+  char *null_ret = get_sdesc_item(null_item);
+
+  cr_assert_eq(ret, new_item->short_desc, "get_sdesc_item() failed to return short description");
```
Comment (lydiafilipe) [inline src/game-state/tests/test_items.c]: Some of these lines are too long - please try to keep them under 80 chars. For strings, if they need to extend multiple lines, you can use a /

**Comment by jessxia** [comment]
Made test print strings into separate lines!

**Comment by jessxia** [comment]
Functions to return the short and long descriptions stored in an item struct (issue #202) was pushed to master. Tests were added to test_items.c.

**Comment by lydiafilipe** [comment]
Issue Score: ✓+

Comments:
Great work!

**Pull request merged**